src/wallet/rpcwallet.cpp

```diff
@@ -1357,13 +1386,23 @@ UniValue listtransactions(const JSONRPCRequest& request)
 {
             RPCHelpMan{"listtransactions",
                 "\nIf a label name is provided, this will return only incoming transactions paying to addresses with the specified label.\n"
-                "\nReturns up to 'count' most recent transactions skipping the first 'from' transactions.\n",
+                "\nReturns up to 'count' most recent transactions skipping the first 'from' transactions.\n"
+                "\nWhen 'paginatebypointer' option is given, the output will have the following changes:\n"
+                "1. Return transactions is ordered by most recent transactions. Though the default does reverse the order after transactions are fetched and clipped.\n"
+                "2. skip argument has no effect. Instead nextpagepointer will be used for pagination.\n"
+                "3. Return value is an object containing, records (array of txs) and nextpagepointer (string),\n",
                 {
                     {"label|dummy", RPCArg::Type::STR, RPCArg::Optional::OMITTED_NAMED_ARG, "If set, should be a valid label name to return only incoming transactions\n"
                           "with the specified label, or \"*\" to disable filtering and return all transactions."},
                     {"count", RPCArg::Type::NUM, /* default */ "10", "The number of transactions to return"},
-                    {"skip", RPCArg::Type::NUM, /* default */ "0", "The number of transactions to skip"},
+                    {"skip", RPCArg::Type::NUM, /* default */ "0", "The number of transactions to skip. If \"usesnextpagepointer\" option is true, skip won't have an effect."},
```

Comment:
I wonder if we should just accept `options` as the 2nd argument in place of `skip`...

Reply:
That will break userland. Or we could have different behaviour depending if it's a number or JSON object?

Reply:
Yes, that's what I mean. We already can have multiple names for the same position.

Reply:
https://github.com/bitcoin/bitcoin/pull/19443#discussion_r449752009

An argument can have aliases but type if same.